internal/services/assets/assets_pubnet.go

```diff
@@ -2,15 +2,27 @@ package assets
 
 import "github.com/stellar/stellar-disbursement-platform-backend/internal/data"
 
+// EURCAssetCode is the code for the EURC asset for pubnet and testnet
+const EURCAssetCode = "EURC"
+
 // USDCAssetCode is the code for the USDC asset for pubnet and testnet
 const USDCAssetCode = "USDC"
 
 // XLMAssetCode is the code for the XLM asset for pubnet and testnet
 const XLMAssetCode = "XLM"
 
+// EURCAssetIssuerPubnet is the issuer for the EURC asset for pubnet
+const EURCAssetIssuerPubnet = "GDHU6WRG4IEQXM5NZ4BMPKOXHW76MZM4Y2IEMFDVXBSDP6SJY4ITNPP2"
```

Comment:
https://stellar.expert/explorer/public/asset/EURC-GDHU6WRG4IEQXM5NZ4BMPKOXHW76MZM4Y2IEMFDVXBSDP6SJY4ITNPP2

Reply:
The official website promotes it here: https://www.circle.com/en/eurc